internal/server/response.go

```diff
@@ -34,22 +35,22 @@ func writeResponse(w http.ResponseWriter, v any, status int) error {
 }
 
 // writeError is a helper function that logs any error and writes it back as an http response
-func writeError(w http.ResponseWriter, status int, message string) {
+func writeError(w http.ResponseWriter, status int, message string, err error, logger *zap.Logger) {
```

Comment:
The docstring of this function can benefit from some clarification now that it got new parameters.
The error itself is not written to the http response (and it shouldn't be) only the message is.

Reply:
Ok I adapted the docstring